<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>35</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QLabel" name="display_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiCondensed</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="display_output">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiCondensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="button_3">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button_2">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button_1">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="button_multiply">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#EEA10C;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="button_minus">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#EEA10C;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="button_0">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button_decimal">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="button_8">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="button_7">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button_4">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="button_6">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="button_9">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="button_plus">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#EEA10C;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_5">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#2A322E;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="button_divide">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#EEA10C;
color:white;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="button_clear">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QPushButton" name="button_equal">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;
border-radius:50px;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
